src/Admin/Ajax.php

```diff
@@ -149,6 +143,7 @@ public function handle_refresh_keys() {
 			wp_send_json_error(
 				array(
 					'message' => __( 'Your account does not support LIVE mode at the moment, it must be validated first. If your account has already been validated, please log out and log in again.', 'payplug' ),
+					'still_inactive' => true
```

Comment:
do you need this?

Reply:
Yes this one come after the user try to check if his account is still inactive or it is active, if it still inactive an error message wil show in the vue side (just under the header block), else the user will be able to use live mode.